src/images/common/Containerfile

```diff
@@ -19,7 +19,7 @@ FROM stagex/openssl:3.0.12@sha256:8e3eb24b4d21639f7ea204b89211d8bc03a2e1b729fb11
 # issue, we should upgrade this again.
 FROM stagex/pcsc-lite:sx2024.03.0@sha256:e720e1795706c7c8c1db14bf730b10521e3ff42e4bed90addc590f7446aac8af AS pcsc-lite
 FROM stagex/pkgconf:1.6.3@sha256:05b4ac6b931cd81fe5aff48c5a77bb9ae472e2bfbce3cc7d4698c670712d4409 AS pkgconf
-FROM stagex/rust:1.81.0@sha256:b7c834268a81bfcc473246995c55b47fe18414cc553e3293b6294fde4e579163 AS rust
+FROM stagex/pallet-rust:1.88.0@sha256:81c39a3b42b0336e439aee3c90e9e1e4da1d32425451e7222198a978eb8c2623 AS rust
```

Comment:
I think the equivalent is "core-rust", not "pallet-rust"? Pallet rust has core-rust and a bunch of other things: https://stagex.tools/packages/pallet/rust/